deployment/stacks/ui-auth.ts

```diff
@@ -262,10 +262,12 @@ export function createUiAuthComponents(props: CreateUiAuthComponentsProps) {
     "REGIONAL"
   ).webAcl;
 
-  new wafv2.CfnWebACLAssociation(scope, "CognitoUserPoolWAFAssociation", {
-    resourceArn: userPool.userPoolArn,
-    webAclArn: webAcl.attrArn,
-  });
+  if (webAcl) {
```

Comment:
```suggestion
  if (!isLocalStack) {
```

Reply:
Well...  I don't like what I did, but I don't like that either, I don't want to do either.  So what happened is, we don't have WAF with localstack because we don't have pro.  We can either if every place we call the WAF construct or I thought it would be more efficient to just make the WAF construct do nothing when its localstack...  well...  then the WAF construct's properties need to have an undefined possibility...  that's why this if (webAcl) needs to be here to make sure its not undefined.  If we don't have to add the undefined potential, we don't need to add this if.  It may be better to huddle to discuss.